github/issues.go

```diff
@@ -44,6 +44,7 @@ type Issue struct {
 	Repository       *Repository       `json:"repository,omitempty"`
 	Reactions        *Reactions        `json:"reactions,omitempty"`
 	Assignees        []*User           `json:"assignees,omitempty"`
+	ClosedBy         *User             `json:"closed_by,omitempty"`
```

Comment:
I will the reordering soon.

BTW, I am relatively new to the `generate` stuff but it seems to be ignoring the field with `User` type? After calling generate for adding a boolean field(like locked), I saw a new getter was generated but there was no change after adding a `User` field. Is it supposed to be so?